pokemongo_bot/cell_workers/drop_items_worker.py

```diff
@@ -0,0 +1,135 @@
+import random
+
+from pokemongo_bot.human_behaviour import sleep
+from pokemongo_bot.item_list import Item
+from pokemongo_bot import logger
+
+class DropItemsWorker(object):
+    def __init__(self, bot, items_count=0, max_storage=100, items={}):
+        self.bot = bot
+        self.api = bot.api
+        self.items_count = items_count
+        self.max_storage = max_storage
+        self.items = items
+
+    def check(self):
+        self.api.get_player()
+        response_dict = self.api.call()
+        self.max_storage = response_dict['responses']['GET_PLAYER']['player_data']['max_item_storage']
+        self.items_count = self.bot.get_inventory_count('item')
+        self.items = self.bot.current_inventory()
+        self.work()
+
+    def work(self):
+        if (self.items_count*100)/self.max_storage > 80:
```

Comment:
i think if the inventory greater than 80% then start cleaning, but it's not a final variable